Snakefile

```diff
@@ -79,6 +79,9 @@ else:
     paired_string = ''
 sample_names = sample_reads.keys()
 
+# also read in desired confidence threshold for Kraken
+confidence_threshold = config['confidence_threshold']
```

Comment:
To stop this from breaking previous config files, add something like 
```
if not 'confidence_threshold' in config: 
    config['confidence_threshold'] = 0.0
```

Reply:
Got it, good plan!